agr_literature_service/api/crud/reference_crud.py

```diff
@@ -304,6 +317,8 @@ def show(db: Session, curie: str, http_request=True):  # noqa
         reference_data["cross_references"] = cross_references
         # del reference_data["cross_reference"]
 
+    # So this wierd we check reference.mod_reference_type BUT
```

Comment:
where do you see this ?  It looked that way in the PR when some lines were collapsed, but expanding it, new lines 322-328 seems to be about MRT and 334-342 about MCA

Reply:
Yep commented at the wrong line I will correct comment and where it is too:-
    # So thisis wierd, we check reference.mod_corpus_association BUT
    # use reference_data["mod_corpus_association"]
    if reference.mod_corpus_association:
        for i in range(len(reference_data["mod_corpus_association"])):